### PR TITLE
Updated to 0.12 version

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Fixes https://github.com/open-ce/open-ce/issues/166.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Comment by npanpaliya** [comment]
I've updated TFP to 0.12.0-rc0 as of now. Will keep the PR open until final release is tagged.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,12 +9,12 @@ package:
...
 source:
   git_url: https://github.com/tensorflow/probability.git
-  git_rev: v{{ version }}
+  git_rev: v{{ version }}-rc0
   patches:
     - 0002-revert-README-addition.patch
 
 build:
-  number: 1
+  number: 2
```
Comment (bnemanich) [inline recipe/meta.yaml:17]: Build number should be 1.

Comment (jayfurmanek) [reply recipe/meta.yaml]: right, since the main version number updated, we keep the build number at 1.

**Comment by jayfurmanek** [comment]
Looks good for now

**Review by jayfurmanek — APPROVED** [review]
Approving, but dont merge until we have the final update

**Pull request merged**